massa-proto/proto/massa/api/v1/versioning.proto

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2023 MASSA LABS <info@massa.net>
+
+syntax = "proto3";
+
+package massa.api.v1;
+
+option csharp_namespace = "Com.Massa.Api.V1";
+option go_package = "github.com/massalabs/massa/api/v1;v1";
+option java_multiple_files = true;
+option java_package = "com.massa.api.v1";
+option objc_class_prefix = "GRPC";
+option php_namespace = "Com\\Massa\\Api\\V1";
+option ruby_package = "Com::Massa::Api::V1";
+option swift_prefix = "GRPC";
+
+// Entry for GetMipStatusResponse
+message MipStatusEntry {
+  // Mip info
+  MipInfo mip_info = 1;
+  // state id
+  ComponentStateId state = 2;
+}
+
+// Same as MipInfo struct in versioning package
+message MipInfo {
+  string name = 1;
+  uint32 version = 2;
```

Comment:
To be sure that we'll be canonical, we always use `fixed32` and `fixed64` and uint variant

List of supported type in Prost: 
https://github.com/tokio-rs/prost